src/blocks.js

```diff
@@ -35,6 +35,13 @@
     "author_name": "Dhanush Nehru",
     "author_github_url": "https://github.com/DhanushNehru"
   },
+  {
```

Comment:
The block.js file should not be edited directly. Please remove and create a meta.json file (see the readme for more information)